### PR TITLE
Es6 version of Awesome Books

### PR DESCRIPTION
**Pull request by harshitajain06**
Added the code according to ES6 in Awesome Books, added files :
- Book.js
- DateTime.js
- Luxon.js
- MenuAdd.js
- MenuList.js
- MenuContact.js

### CODE REVIEW

**Comment by ohyajk** [comment]
👍

File: index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+setTime();
+
+const show = new TitleAuthor();
+show.showBook();
+
+const submit1 = document.querySelector('#submit_button');
+
+submit1.addEventListener('click', () => {
+  show.submit();
+  // alert('Books Added');      // eslint-disable-line
```
Comment (aimalamiri) [inline index.js]: Good job, but please make sure to remove the commented code lines from your codebase, and keep your repository clean.

File: modules/book.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const storedTitleAuthor = JSON.parse(localStorage.getItem('students'));
+    if (!storedTitleAuthor) {
+      titleAuthorArray.push(title_author);
+      localStorage.setItem('students', JSON.stringify(titleAuthorArray));
+    } else {
+      storedTitleAuthor.push(title_author);
+      localStorage.setItem('students', JSON.stringify(storedTitleAuthor));
+    }
+
+        location.reload(); // eslint-disable-line
```
Comment (aimalamiri) [inline modules/book.js]: It works, but in order to keep your code professional it's highly recommended to follow the eslint rules, for example in this case you can fix the linter error by adding `window` to the beginning of the `location` like `window.location.reload();` I'd suggest to google for others as well and avoid skipping the linter checks.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @harshitajain06, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The app is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @harshitajain06,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**